dspace_import.py

```diff
@@ -815,12 +861,12 @@ def import_bitstream():
             metadata_bitstream = get_metadata_value(0, i['bitstream_id'])
             if metadata_bitstream:
                 json_p['metadata'] = metadata_bitstream
-            json_p['sizeBytes'] = i['size_bytes']
-            json_p['checkSum'] = {'checkSumAlgorithm': i['checksum_algorithm'], 'value': i['checksum']}
+            json_p['sizeBytes'] = 1748
```

Comment:
this shouldn't go into main

Reply:
It is there because this code will never go to the main. It is only for internal use, as I mentioned in the note before this review. Also, we don't have an asset store; we only have one file that we use to import each bitstream. That's why there are constants.